frontend/public/global.css

```diff
@@ -3,7 +3,8 @@
   --main-fg-color: white;
   --selected-bg-color: #86e3ed;
   --comment-color: #eb8e5b;
-  --modified-color: #dc4e47;
+  --last-modified-color: #dc4e47;
+  --all-modified-color: #fdb44e;
```

Comment:
These colors also should be set in `frontend/src/stores.js` in the default settings object. https://github.com/redballoonsecurity/ofrak/blob/eb0c5d6d2a3942abc11054f2b46f3a99757b37bd/frontend/src/stores.js#L15-L30

The corresponding setting should have a GUI element added in the `frontend/src/SettingsView.svelte` file. https://github.com/redballoonsecurity/ofrak/blob/eb0c5d6d2a3942abc11054f2b46f3a99757b37bd/frontend/src/SettingsView.svelte#L243-L246

And the color should be used in `frontend/src/App.svelte`. https://github.com/redballoonsecurity/ofrak/blob/eb0c5d6d2a3942abc11054f2b46f3a99757b37bd/frontend/src/App.svelte#L160-L165

As part of this change, it's worth adding a comment to this CSS that points a developer to these three places.